### ISSUE TITLE
Throw Exception instead "return;"

### ISSUE THREAD

**Issue by Slayer911** [mark-gerarts/automapper-plus#52]
Please, change this comment to real exception
https://github.com/mark-gerarts/automapper-plus/blob/d5ca462e99deae8c6628d15d5c67ab479ad1b8b9/src/MappingOperation/DefaultMappingOperation.php#L49
When this library is one block of big system, bugs like this is hard detection, if here will be real exception, life will be better,  i think

**Comment by mark-gerarts** [comment]
Yes, you are right. However, throwing an exception here could be a backwards incompatible change. I will definitely incorporate this in the 2.0 release though. Thanks for reporting!

**Comment by Slayer911** [comment]
Thank for great library)
